src/EditorFeatures/Core/Extensibility/Navigation/NavigableItemFactory.DeclaredSymbolNavigableItem.cs

```diff
@@ -53,6 +51,22 @@ public DeclaredSymbolNavigableItem(Document document, DeclaredSymbolInfo declare
                     }
                 });
             }
+
+            private ISymbol FindSymbol()
+            {
+                // Here, we will use partial semantics. We are going to use this symbol to get a glyph, display string,
+                // and potentially documentation comments. The first two should work fine even if we don't have full
+                // references, and the latter will probably be fine. (It wouldn't be if you have a partial type
+                // and we didn't get all the trees parsed yet to know that. In other words, an edge case we don't care about.)
```

Comment:
Cyrus and I discussed this and came to the conclusion he's already described -- in practice we don't think it matters. In our testing of opening Roslyn.sln and doing some stress-test searches, it was fine too.